automation_tools/__init__.py

```diff
@@ -1852,9 +1852,9 @@ def copy_ssh_key(from_host, to_host):
 
 
 def sync_capsule_tools_repos_to_upgrade(admin_password=None):
-    """This syncs capsule and sat-tools repos in Satellite server.
+    """This syncs capsule repos in Satellite server.
 
-    Useful for upgrading Capsule and Client in feature.
+    Useful for upgrading Capsule in feature.
```

Comment:
why is client stuff being dropped here? Capsule does require sat-tools repo too.